packages/nextjs/app/builders/0x9B28C43d4526202c316b9ab0ECCB757C4D9c5155/page.tsx

```diff
@@ -0,0 +1,105 @@
+"use client";
+
+import { useEffect, useState } from "react";
+import Image from "next/image";
+import me from "./me.webp";
+import "./style.css";
+
+export default function Jriyyya() {
```

Comment:
Nitpick: 

For pages, In SE-2 we tend to use arrow functions for page and then export it as default at bottom, checkout `app/page.tsx` for example. 


Just trying to stay consistent. but its completely fine if we use function and default export directly !

Reply:
Done